fivox/fieldFunctor.h

```diff
@@ -81,6 +88,9 @@ FieldFunctor< TImage >::operator()( const TPoint& point, const TSpacing& ) const
     float sum = 0.f;
     for( const Event& event : events )
     {
+        const float eventValue =
+                ( event.value - brion::MINIMUM_VOLTAGE ) * _magnitude;
+
```

Comment:
I don't think we should clamp or ignore any value. It could perfectly be a negative value, as we use floating point numbers during the whole workflow. And actually, the way I see it, subtracting the minimum voltage is also wrong, we should just set the value contained in the report. At the end of the process, at least for the voxelize tool, is where it is rescaled to the output type. I'm not sure about Livre's datasource though.